docker/openproblems/Dockerfile

```diff
@@ -36,11 +17,5 @@ COPY . /usr/src/singlecellopenproblems
 RUN cd /usr/src/singlecellopenproblems && git clean -fxdq
 RUN pip install --no-cache-dir --editable /usr/src/singlecellopenproblems
 
-# Overwrite kernel.json to use system Python install
-COPY ./docker/openproblems/kernelspec.json /usr/local/share/jupyter/kernels/python3/kernel.json
```

Comment:
what was this used for? Doesn't look like a sagemaker requirement to me.

Reply:
kernelspec.json tells sagemaker which python version to use.